### ISSUE TITLE
[Bug] Preferences being reset after reboot

### ISSUE THREAD

**Issue by thejasonparker** [will-stone/browserosaurus#83]
**Issue:** 
When I reboot my MacBook Laptop my saved preferences _(browser order set in Activities)_ are reset to the default list order of - Google Chrome / Firefox / Safari / Copy to Clipboard

The config.json file in Library⁩ ▸ ⁨Application Support⁩ ▸ ⁨Browserosaurus⁩ is being written to when the browser order gets updated (Safari / Google Chrome / Firefox / Copy to Clipboard) but after a reboot this gets changed back to the default order (Google Chrome / Firefox / Safari / Copy to Clipboard).

**Browserosaraus Version :** v4.4.0
**MacOS Mojave :** 10.14.2 (18C54)
**Browsers :** Safari / Chrome / Firefox (latest versions)

Thanks for a really useful application and I hope this helps fix the bug.

**Comment by will-stone** [comment]
Interesting. I'll have to see if I can replicate that some time. Sadly, until #77 if fixed, I probably won't be spending much time on coding Browserosaurus as I'm unable to use it.

**Comment by thejasonparker** [comment]
I idd notice the delay after I updated to Mojave but put it down to that and still use Browserosaraus. Even with the delay and preferences bug I still find it useful :)

**Comment by CedricGatay** [comment]
Hi, the issue you're describing look like the one I've fixed in the following PR https://github.com/will-stone/browserosaurus/pull/81  
Feel free to try this version and tell me if it fixes things as it could always @will-stone to merge it faster.

**Comment by will-stone** [comment]
Can you please tell me if v4.5.0 fixes this?

**Comment by thejasonparker** [comment]
Yep, that seems to have fixed it. I have rebooted 3 times now and my preferences stay as set.

Thanks for fixing things 👍
